src/components/Booking/BookingContextProvider.jsx

```diff
@@ -10,26 +10,35 @@ export function useBookingContext() {
 }
 
 function BookingContextProvider({ children }) {
-  const [selectedDate, setSelectedDate] = useState(null);
+  const [selectedDate, setSelectedDate] = useState(findTodayDate());
   const [weekDaysDetails, setWeekDaysDetails] = useState(null);
-
   const [daySlotMapping, setDaySlotMapping] = useState([]);
+  const [selectedTimeslot,setselectedTimeslot] = useState([]);
+ 
 
   useEffect(() => {
     if (weekDaysDetails && selectedDate) {
+     
       const mappedSlots = weekDaysDetails
         .filter(element => element.days_of_week.includes(format(selectedDate, 'EEEE').toUpperCase().substring(0, 3)))
         .map(element => ({
           slot_group: element.slot_group,
           time: element.time,
+          duration:element.duration,
+          slot_index:element.slot_index
         }));
-        
       setDaySlotMapping(mappedSlots);
     }
   }, [selectedDate, weekDaysDetails]);
+   
+  function findTodayDate(){
+    let date = new Date();
+    return date;
+  }
```

Comment:
Move it date help and remame the function reflect its purpose eg: getTodaysDate 
findTodayDate feels deceptive